data/B2prakash.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "Ved prakash",
+  "type": "personal",
+  "bio": "student , open source contributer ",
+  "tags": [
+    "Open Source",
+    "Javascript",
+    "Typescript",
+    "NextJS",
+    "DevOps",
+    "DevRel"
+  ],
+  "socials": [
+   { "icon": "FaTwitter", "url": "https://twitter.com/VedPrak39212943" },
+   { "icon": "FaGithub", "url": "https://github.com/B2prakash" },
+   { "icon": "FaYoutube", "url": "https://youtube.com/vedprakash2431" }
+  ],
+  "links": [
+     {
+      "group": "Primary socials",
+      "name": "Twitter: Follow me",
+      "url": "https://twitter.com/vedprak39212943",
+      "icon": "FaTwitter",
+      "color": "#00ACEE"
+    },
+    {
+      "group": "Primary socials",
+      "name": "linkedin: connect with me ",
+      "url": "https://linkedin.com/in/vedprakash",
```

Comment:
Not a blocker, but it seems like someone else's profile. If it is not yours, then please update it with yours. 
Rest looks good to me!

![image](https://user-images.githubusercontent.com/88786642/228062059-af410dea-7283-49ac-8614-a2cc10f0bc83.png)